js/menu.js

```diff
@@ -0,0 +1,7 @@
+const toggleMenuButton = document.getElementById('toggle-menu');
+const menu = document.getElementById('menu');
+
+toggleMenuButton.addEventListener('click', () => {
+  toggleMenuButton.classList.toggle('active');
+  menu.classList.toggle('active');
+});
```

Comment:
According to the requirements:

![image](https://user-images.githubusercontent.com/57421823/156000572-969ed605-3853-4f7f-baca-35ecfbf18fab.png)

When I click on a menu item,  if I scroll the page up I can see the menu is still there.

Therefore:

- [ ] Please, make sure the menu correctly disappears when a user clicks or taps any menu item.

That way, the user experience will be improved a lot! 💪